packages/salesforcedx-vscode-soql/src/commands/soqlFileCreate.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { EDITOR_VIEW_TYPE } from '../constants';
+import { telemetryService } from '../telemetry';
+
+export async function soqlOpenNew() {
+  telemetryService.sendCommandEvent('soql_builder_open_new', process.hrtime());
+
+  if (vscode.workspace) {
+    // create untitled file
+    const newFileUri = vscode.Uri.file(
+      path.join(vscode.workspace.rootPath!, 'untitled.soql')
```

Comment:
I wonder if until we give the customers the option to save results where they want we should be sticking new `.soql` files in the same place we expect them to be in an sfdx-project? 
`scripts/soql` as a default? Just to keep things in the same place.
Though if we are opinionated now, that might mean more work to undo it in the future...

Just putting it out there, either way.

Reply:
Most of the artifacts we deal with in a project have designated locations, which makes means that creating an Apex class or an LWC component requires only a name. If we are _not_ opinionated, that means we have to gather a folder _and_ a filename from the customer. This is one of the reasons I like the untitled flow. It gets them in the editor quickly, and they may not even care to have a file at the end if they're just there to build and test.

Reply:
That makes sense, I also like this part too
>  It gets them in the editor quickly, and they may not even care to have a file at the end if they're just there to build and test.